### PR TITLE
Add support for help_text field parameter as docstring for django model fields.

### LINKED ISSUE #127: Add support for help_text field parameter as description with django model fields.
**Is your feature request related to a problem? Please describe.**
Now django support uses only `verbose_name` for field description. It would be great to take advantage of `help_text` parameter which seems to be the best place to place long description about what exactly given field does.

According to the docs `help_text` is the place where we should place documentation about the field ([docs](https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.Field.help_text)) rather than `verbose_name` which is actually only longer name for the field. 

**Describe the solution you'd like**
Concat verbose_name with help_text when help_text is specified. Maybe like so:
```python
f"{field.verbose_name} - {field.help_text}"
```

**Describe alternatives you've considered**
As the `verbose_name` should be concise I'm using doctsring with Attributes which is copy paste from fields and a lot of manual work which is unneeded.

**Additional context**
This is directly related to #39

### PR DESCRIPTION
**Pull request by mabugaj**
Here are added features:

- support for `help_text` parameter of django model field
- casting docstring to string type, which solves the problem of serializing `Proxy` objects in case of lazy translation

This is the final result with and without `help_text`:

image

this closes #127

### CODE REVIEW

File: src/pytkdocs/loader.py [hunk 1]
```diff
@@ -741,11 +741,19 @@ def get_django_field_documentation(node: ObjectNode) -> Attribute:
         if prop.blank:
             properties.append("blank")
 
+        # set correct docstring based on verbose_name and help_text
+        # both should be converted to str type in case lazy translation
+        # is being used, which is common scenario in django
+        if prop.help_text:
+            docstring = "`{}`: {}".format(prop.verbose_name, prop.help_text)
```
Comment (pawamoy) [inline src/pytkdocs/loader.py:748]: Lets not assume Markdown:
```suggestion
            docstring = f"{prop.verbose_name}: {prop.help_text}"
```

File: src/pytkdocs/loader.py [hunk 2]
```diff
@@ -741,11 +741,19 @@ def get_django_field_documentation(node: ObjectNode) -> Attribute:
...
+        else:
+            docstring = "{}".format(prop.verbose_name)
```
Comment (pawamoy) [inline src/pytkdocs/loader.py:750]:
```suggestion
            docstring = prop.verbose_name
```

Comment (mabugaj) [reply src/pytkdocs/loader.py:750]: If You won't cast it to string then You get serializer error with Proxy object that can't be serialized. Of course that only happens when `verbose_name` uses lazy translation (which is very common practice in django).

Comment (pawamoy) [reply src/pytkdocs/loader.py:750]: Ah, my bad! Then a simple cast with str should suffice: `docstring = str(prop.verbose_name)`, no?

Comment (mabugaj) [reply src/pytkdocs/loader.py:750]: I would go with format or f string which seems to be safer solution but str() should be ok in most cases.

Comment (pawamoy) [reply src/pytkdocs/loader.py:750]: I don't think there's a difference, as f-strings and `.format` just call `__str__` on the object (happy to be proven wrong).

Comment (mabugaj) [reply src/pytkdocs/loader.py:750]: You seems to be right. Somehow I've thought that format should do some magic with custom types but apparently it don't. So we can safely go with `docstring = str(prop.verbose_name)`.

**Review by pawamoy — APPROVED** [review]
OK, good to go!

**Pull request merged**